wercker.yml

```diff
@@ -222,6 +222,26 @@ build:
           echo "build is skipped ..."
         fi
 
+  - script:
+      name: Imports - guava ....
```

Comment:
name should be "NoErrorTest - guava imports"

Reply:
done